src/lib/transition_router/initial_validator.ml

```diff
@@ -199,6 +199,6 @@ module Make (Inputs : Transition_frontier.Inputs_intf) = struct
                 ; ( "transition"
                   , External_transition.to_yojson
                       (With_hash.data transition_with_hash) ) ]
-              !"Failed to validate transition from $peer" )
+              "Failed to validate transition from peer $peer" )
```

Comment:
This whole message is redundant actually. `handle_validation_error` logs and calls the trust system, which logs as well.